ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -158,15 +160,6 @@ describe('HelpSidebar', () => {
     expect(deleteSpy).toHaveBeenCalled();
   });
 
-  it('should call share method when clicked', async() => {
```

Comment:
instead of deleting the test, could you check if the modal exists after the button is clicked? even if empty as mocked out above